kratos/tests/cpp_tests/strategies/schemes/test_residual_based_adjoint_bossak_scheme.cpp

```diff
@@ -262,14 +287,94 @@ class PrimalElement : public Element
         rDampingMatrix(1, 1) = damping;
         rDampingMatrix(1, 0) = -damping;
     }
+};
 
```

Comment:
In ``CalculateRightHandSide`` of the element the factor of "3" is missing in some parts of the formulas within lines 269/270.

If here the factor of "3" is added it is also necessary within ``CalculateSensitivityMatrix``